packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.tsx

```diff
@@ -224,6 +228,7 @@ export class DetailsRow extends BaseComponent<IDetailsRowProps, IDetailsRowState
         style={ { minWidth: viewport ? viewport.width : 0 } }
         aria-selected={ isSelected }
         allowFocusRoot={ true }
+        isInnerZoneKeystroke={ (ev: React.KeyboardEvent<HTMLElement>) => (this.props.innerZoneFocusable ? ev.which === getRTLSafeKeyCode(KeyCodes.down) : false) }
```

Comment:
I'm not completely sure why we need this here. In fact, It feels really wrong, given that rows are hosted in a vertical zone. And it is a performance concern to nest FocusZones in a DetailsRow. This a hot spot.

If you're simply trying to move focus to vertically aligned items within a zone, this should be possible now with FocusZone as is, since now horizontal will still move up and down through items that are organized correctly in the DOM.

Example, each row in this codepen has 3 vertically aligned buttons, but arrowing left and right, you can still move focus to them:

https://codepen.io/dzearing/pen/XRvXoL